### PR TITLE
713 multiple notification

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- Collaboration user manager create must be able to be sent to the same user multiple times (for different requests) 
- Reworked the configuration for this so that the notification code that knows this configuration

---

### CODE REVIEW

File: app/modules/collaborations/resources.py [hunk 1]
```diff
@@ -79,6 +79,9 @@ def post(self, args):
         from app.modules.users.models import User
 
         req = json.loads(request.data)
+        if not isinstance(req, dict):
```
Comment (hwindsor) [inline app/modules/collaborations/resources.py:82]: Hit this bug while testing so fixed it

File: app/modules/notifications/models.py [hunk 2]
```diff
@@ -129,11 +129,13 @@ class NotificationChannel(str, enum.Enum):
         'email_template_name': 'collaboration_manager_create',  # Not yet written
         'email_digest_content_template': 'collaboration_manager_create_digest',
         'mandatory_fields': {'collaboration_guid', 'user1_name', 'user2_name'},
+        'allow_multiple': True,
     },
     NotificationType.collab_manager_revoke: {
         'email_template_name': 'collaboration_manger_revoke',  # Not yet written
         'email_digest_content_template': 'collaboration_manager_revoke_digest',
         'mandatory_fields': {'collaboration_guid', 'user1_name', 'user2_name'},
+        'allow_multiple': True,
```
Comment (hwindsor) [inline app/modules/notifications/models.py:138]: @naknomum Have a look at the merge notifications. Do you need to update them to allow multiple merge requests from the same user for different individuals?
Hint, I think you do.

**Review by colinwkingen — APPROVED** [review]
LGTM

File: app/modules/notifications/models.py (hunk 2 above)
Comment (naknomum) [reply app/modules/notifications/models.py:138]: indeed it seems like i do.   created [DEX-730](https://wildme.atlassian.net/browse/DEX-730) for this, assigned to me.

File: app/modules/notifications/models.py [hunk 3]
```diff
@@ -166,7 +168,6 @@ class NotificationBuilder(object):
     def __init__(self, sender, multiple=False):
```
Comment (hwindsor) [inline app/modules/notifications/models.py]: @naknomum pointed out that multiple was no longer needed, now droppped

**Review by naknomum — APPROVED** [review]
looks good.

**Pull request merged**